Migrate to Astro - Guide Stubs (#2232)
withastro/docs: src/pages/en/guides/migrate-to-astro/from-hugo.mdx

---
title: Migrating from Hugo
description: Tips for migrating an existing Hugo project to Astro
layout: ~/layouts/MigrationLayout.astro
stub: true
framework: Hugo
---

[Hugo](https://gohugo.io) is an open-source static site generator built on Go.

## Key Similarities between Hugo and Astro

Hugo and Astro share some similarities that will help you migrate your project:

- Hugo and Astro are both modern static-site generators, ideally suited to content-focused sites like blogs.

> - Hugo and Astro both allow you to author your content in Markdown. However, Hugo allows you to write frontmatter in YAML, TOML or JSON and recognizes several special properties while allowing for custom properties. Astro recognizes YAML frontmatter, but has very few reserved frontmatter properties. Almost all of your frontmatter will be user-defined. Even though many of your existing Hugo frontmatter properties will not be "special" in Astro, this metadata can be accessed and used by Astro components. So, you can continue to use your existing Markdown files and frontmatter values.

Reviewer: ```suggestion
- Hugo and Astro both allow you to author your content in Markdown. However, Hugo includes several special frontmatter properties and allows you to write frontmatter in YAML, TOML or JSON. Even though many of your existing Hugo frontmatter properties will not be "special" in Astro, you can continue to use your existing Markdown files and frontmatter values.
```
